src/assets/styles/PrimaryButton.css

```diff
@@ -1,3 +1,49 @@
+html {
+  font-size: 10px;
+}
+
+.registerForm{
```

Comment:
RegisterForm, addPet and updateUser are almost the same. create one selector add attributes in common

Reply:
Thanks leonel. I refactor and now there is less noise in the code. No need for that duplicated styles in the classes!